### PR TITLE
Initial docs and some basic custom validations

### PR DESCRIPTION
**Pull request by agch-dev**
This PR adds the basic structure of the library, docs, and some custom validations

Some links to files in the PR might not work because they reference the master branch, so once merged they should work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
+<p align="center"><img src="resources/logo.png" /></p>
+
 # @rootstrap/validate
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/maintainability)](https://codeclimate.com/github/rootstrap/validate/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/test_coverage)](https://codeclimate.com/github/rootstrap/validate/test_coverage)
 
 [Description]
```
Comment (felipegarcia92) [inline README.md]: I guess you want to remove this one 😄

**Review by felipegarcia92 — COMMENTED** [review]
Great work 👏

File: README.md (hunk 1 above)
Comment (agch-dev) [reply README.md]: 😅

Comment (agch-dev) [reply README.md]: Fixed!

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/maintainability)](https://codeclimate.com/github/rootstrap/validate/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/test_coverage)](https://codeclimate.com/github/rootstrap/validate/test_coverage)
 
-[Description]
-Add the name and description of the library.
+Have you ever had to write your own custom validations? It can be a bit confusing, specially if you have never done it before. Don't worry, we are here to help.
+
+This library extends the popular library [validate.js](https://validatejs.org) and adds some custom validations that have proven to be very useful and quite commonly needed. Don't reinvent the wheel, next time you need to write a custom validation, check if someone hasn't implemented that custom validation already.
+
+Aside form common custom validations, this library will also serve as a gallery of all the custom validations that we have had to implement at Rootstrap, although some might not be included out of the box, we intend to showcase them in case you need them. [Here is a link]() to said gallery
```
Comment (mrubi-rootstrap) [inline README.md]: typo in form maybe?

Comment (agch-dev) [reply README.md]: hehe, my achilles heel I swap those two words all the time.

Comment (mrubi-rootstrap) [reply README.md]: oh, I got it. It's validating a form. Nevermind

Comment (agch-dev) [reply README.md]: but you are right! it should be `from` instead of `form`

**Review by fernandatoledo — APPROVED** [review]
🚀 👏

**Review by felipegarcia92 — APPROVED** [review]

**Pull request merged**